android/sdl_android/src/androidTest/java/com/smartdevicelink/test/rpc/datatypes/DoorStatusTests.java

```diff
@@ -0,0 +1,64 @@
+package com.smartdevicelink.test.rpc.datatypes;
+
+import com.smartdevicelink.marshal.JsonRPCMarshaller;
+import com.smartdevicelink.proxy.rpc.DoorStatus;
+import com.smartdevicelink.proxy.rpc.Grid;
+import com.smartdevicelink.proxy.rpc.RoofStatus;
+import com.smartdevicelink.proxy.rpc.enums.DoorStatusType;
+import com.smartdevicelink.test.JsonUtils;
+import com.smartdevicelink.test.TestValues;
+import com.smartdevicelink.test.Validator;
+
+import junit.framework.TestCase;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.Hashtable;
+import java.util.Iterator;
+
+public class DoorStatusTests extends TestCase {
+    private DoorStatus msg;
+
+    @Override
+    public void setUp() {
+        msg = new DoorStatus(TestValues.GENERAL_GRID, TestValues.GENERAL_DOOR_STATUS_TYPE);
+    }
+
+    public void testRpcValues() {
+        // Test Values
+        Grid location = msg.getLocation();
+        DoorStatusType status = msg.getStatus();
+
+        // Valid Tests
+        assertEquals(TestValues.MATCH, TestValues.GENERAL_GRID, location);
```

Comment:
@kostyaBoss You can add this line below line 34:

`assertEquals(TestValues.MATCH, TestValues.GENERAL_DOOR_STATUS_TYPE, status);`